auto-generated/vector-crypto/llvm-api-tests/vaesdf.c

```diff
@@ -0,0 +1,87 @@
+// REQUIRES: riscv-registered-target
+// RUN: %clang_cc1 -triple riscv64 -target-feature +v -disable-O0-optnone \
```

Comment:
Can you also add `-target-feature +zvl256b` since we have test cases with `LMUL=1/2` such as `__riscv_vaesdf_vv_u32mf2`, if not added, it would cause an error.